var/spack/repos/builtin/packages/libfive/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Libfive(CMakePackage):
+    """libfive is a software library and set of tools for solid modeling."""
+
+    homepage = "https://libfive.com"
+    git      = "https://github.com/libfive/libfive.git"
+
+    # https://libfive.com/download/ recommends working from the master branch
+    version('develop', branch='master')
```

Comment:
You can call this version `master` now that #1983 has been merged.

Reply:
@adamjstewart Done.